atom/browser/ui/accelerator_util.cc

```diff
@@ -109,6 +109,10 @@ bool StringToAccelerator(const std::string& description,
         modifiers |= ui::EF_SHIFT_DOWN;
     } else if (tokens[i] == "ctrl" || tokens[i] == "control") {
       modifiers |= ui::EF_CONTROL_DOWN;
+#if defined(OS_WIN)
```

Comment:
The `Windows key or Command key` proper name seems to be `Super Key`.
http://en.wikipedia.org/wiki/Super_key_%28keyboard_button%29

Maybe the shortcut should be named `super` instead of `win` ?